### ISSUE TITLE
Parse Resume

### ISSUE THREAD

**Issue by amelinfo** [pbearne/Bullhorn2WP#4]
Hi, i want to thank you for this great plugin , but i want to ask you how to use parseResume() function alone, that mean: the user upload his resume and i want to parse it and redirect him to a form which contain data extracted from the file . i wish you can understand me.

Thanx,

**Comment by pbearne** [comment]
Yes that can be done 
have look at the function 
**https://github.com/pbearne/Bullhorn2WP/blob/master/bullhorn-cv.php#L393**

if cab take a array of file data or if you the file have been just uploaded it will find them 
 parseResume( $local_files )

Note it needs the uploaded file to be called 'resume'
$local_file = $local_files['resume']['tmp_name'];

after you have had the you form filled you can use the createcanidate functions to push to bullhurn

Paul

**Comment by amelinfo** [comment]
I did it , Thank you 👍

**Comment by pbearne** [comment]
are you using the dev branch from here on git hub it has local saving for the CV uploads?

Note. I have an ongoing bug with the saving of the CV to bullhorn as attachment (this is want is holding the release up)

I will take pull request if spot any problems ;-)

**Comment by amelinfo** [comment]
yes, i found cv directory and for the moment i commented the function that create this folder and just used the rest call of parsing. "i need to add many modifications (like the user can apply just one time to an offer  ...) ". I am waiting for your next commit and if i find an issue i ill inform you ;-)
